test/extensions/voting-rep.js

```diff
@@ -1525,6 +1525,22 @@ contract("Voting Reputation", (accounts) => {
       const pastVote = await voting.getExpenditurePastVote(slotHash);
       expect(pastVote).to.eq.BN(REQUIRED_STAKE);
     });
+
+    it("motions with the special NO_ACTION signature do not require (and cannot be) executed, and go straight to that state", async function () {
+      const action = "0x12345678";
+
+      await voting.createMotion(1, UINT256_MAX, ADDRESS_ZERO, action, domain1Key, domain1Value, domain1Mask, domain1Siblings);
+      motionId = await voting.getMotionCount();
+
+      await voting.stakeMotion(motionId, 1, UINT256_MAX, YAY, REQUIRED_STAKE, user0Key, user0Value, user0Mask, user0Siblings, { from: USER0 });
+
+      await forwardTime(STAKE_PERIOD, this);
+
+      await checkErrorRevert(voting.finalizeMotion(motionId), "voting-rep-motion-not-finalizable");
+
+      const motionState = await voting.getMotionState(motionId);
+      expect(motionState).to.eq.BN(EXECUTED);
```

Comment:
Just noting how in the tests we call these `EXECUTED` while in the contracts they're `Finalized`, but perhaps not a big deal.